--- ../linux/drivers/usb/dwc2/gadget.c	2017-01-13 14:49:00.711208323 -0500
+++ gadget_orig.c	2017-01-13 13:47:56.293600033 -0500
@@ -1207,15 +1207,6 @@
 		ctrl->bRequestType, ctrl->bRequest, ctrl->wValue,
 		ctrl->wIndex, ctrl->wLength);
 
-        /* Prints setup requests */
-        printk("USB DWC2 REQ %02x %02x %04x %04x %04x\n",
-                ctrl->bRequestType,
-                ctrl->bRequest,
-                ctrl->wValue,
-                ctrl->wIndex,
-                ctrl->wLength);
-
-
 	if (ctrl->wLength == 0) {
 		ep0->dir_in = 1;
 		hsotg->ep0_state = DWC2_EP0_STATUS_IN;
